operator/cmd/mesh/manifest-common.go

```diff
@@ -167,36 +45,24 @@ func ignoreError(stderr string) bool {
 	return trimmedStdErr == ""
 }
 
-// fetchInstallPackageFromURL downloads installation packages from specified URL.
-func fetchInstallPackageFromURL(mergedIOPS *v1alpha1.IstioOperatorSpec) error {
```

Comment:
I suggest we keep this for upgrade to compare with older version manifests, at least in 1.5. 
In 1.6 we have dual control plane upgrade, until then we can remove fetching package from url.

Reply:
Are you saying we should keep this code or it's good that it's being deleted? It seemed to me it's not plumbed in anyway so there's not much point keeping it - we can always restore it if we want the function back.